lib/coffeescript/parser.js

```diff
@@ -1,4 +1,4 @@
-/* parser generated by jison 0.4.17 */
+/* parser generated by jison 0.4.13 */
```

Comment:
Somehow you don’t have the latest version of Jison installed. Please `npm install`.